src/Todoist.Net/Models/CommandResult.cs

```diff
@@ -0,0 +1,19 @@
+namespace Todoist.Net.Models
+{
+    internal class CommandResult
+    {
+        private CommandResult()
+        { }
+
+        public CommandError CommandError { get; set; }
```

Comment:
Do we need the public setter here?

Reply:
No, in fact, we don't even need an internal setter. 
I will make it private as we rely on the static methods for initialization.